WinEventParser/main.py

```diff
@@ -20,38 +20,38 @@ def admin():
         return False
 
 
+def is_valid_char(char):
+    """Function to check if each char of timestamp are valid"""
+    acceptable_chars = ['-', ' ', ':']
+    return char.isdigit() or char in acceptable_chars
+
+
+def get_chars_to_add(mode):
```

Comment:
Function is great!
Let's rename a bit so others can immediately understand what kind of characters are being added.

Reply:
something like get_string_to_add_to_timestamp() or get_completion_of_timestamp() ?